cmd/minishift/cmd/addon/addon_install.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 const (
-	unspecifiedSourceError   = "You need to specify the source for the add-on."
+	unspecifiedSourceError   = "You must specify the source directory path of the add-on."
```

Comment:
@hferentschik I think we should make the change `You must specify the source of the add-on.` when we add the code to implement other source types.